README.md

```diff
@@ -46,6 +46,8 @@ WHERE {{keyColumn}} > {{position.last_processed_val}}
 FETCH NEXT {{batchSize}} ROWS ONLY;
 ```
 
+This behavior is enabled by default, but can be turned off by adding `"snapshot": "false"` to the Source configuration.
```

Comment:
maybe it is must be the last line on this part.
What do you think about it

Reply:
I thought about it but decided to put it in the last line before the CDC information.
But if you think it's better, I can update.

Reply:
lets keep like this